source/reference/modules/deployer/index.rst

```diff
@@ -355,6 +354,122 @@ the steps described in :ref:`debugging-deployer-issues`
   Changing or deleting a processed commit file could cause unchanged files to be indexed again and
   it should be done as a last resort in case of errors.
 
+^^^^^^^^^^^^^^^^^
+Document Indexing
+^^^^^^^^^^^^^^^^^
+Crafter Deployer indexes documents via the following configured items:
+
+- A full text index of any document that has a mimetype that matches the configured list of mimetypes.
+- Indexing of any remote document that matches the configured list of remote documents pattern
+- Indexing of jacketed documents with anything that matches the configured pattern.
+
+Note that indexing of documents in authoring and indexing of documents in delivery each have their own configuration.
+
+"""""""""
+Mimetypes
+"""""""""
+The list of supported mimetypes determines what's considered a document that should be full-text indexed.
```

Comment:
The following is the default list of MIME-types with full-text-search indexing enabled.